test/unit/test_buf.c

```diff
@@ -154,27 +154,54 @@ bool test_r_buf_mmap(void) {
 	write (fd, content, length);
 	close (fd);
 
-	b = rz_buf_new_mmap (filename, RZ_PERM_RW);
+	b = rz_buf_new_mmap (filename, O_RDWR, 0);
```

Comment:
Might be out of scope for this PR but I think that `rz_sys_open` should accept `RZ_PERM` and use your `rizinperms2open` in the same manner as `rz_io_def_mmap_create_new_file` to keep things consistent. 
Especially when `O_CREAT` is implicitly toggled in `RZ_PERMS` for WR while having to set it explicitly with functions that accept `O_*` perms.

Reply:
I agree. I started doing that but it required many changes all over the place so I ended up keeping it like this for now.